public-node_connection_info.config

```diff
@@ -0,0 +1 @@
+[
```

Comment:
Is this file needed?

Reply:
no should be in gitignore, will sort that